src/Model/Contact.php

```diff
@@ -1232,6 +1244,9 @@ public static function photoMenu(array $contact, int $uid): array
 				'pm'       => [DI::l10n()->t('Send PM'), $pm_url, false],
 				'follow'   => [DI::l10n()->t('Connect/Follow'), $follow_link, true],
 				'unfollow' => [DI::l10n()->t('Unfollow'), $unfollow_link, true],
+				'mention'  => [DI::l10n()->t('Mention'), DI::l10n()->t('Post to group'), $mention_url, false],
+				'group'    => [DI::l10n()->t('View group'), $group_link, $contact['forum'], true],
+				'self'     => [$contact['self'] ?? false, true],
```

Comment:
Why does this template variable have two potentially different values given that you only query the first in the template?
```suggestion
				'self'     => $contact['self'] ?? false,
```
This would also be better placed in  `src/Content/Widget/Hovercard.php` under the `'$profile'` template variable.

Reply:
I don't know why, but if i set it only with one value, it is empty, when i query it's content...

Reply:
Since it's a boolean value, it shouldn't produce any output when you try to display it in the template.